packages/platform-browser/src/errors.ts

```diff
@@ -12,6 +12,7 @@
 export const enum RuntimeErrorCode {
   // Hydration Errors
   UNSUPPORTED_ZONEJS_INSTANCE = -5000,
+  HYDRATION_CONFLICTING_FEATURES = -5001,
```

Comment:
We don't have a doc file for this error. Negative error code is a flag to add a link to the error message. 

```suggestion
  HYDRATION_CONFLICTING_FEATURES = 5001,
```

Reply:
Updated